src/components/BulkEditList/BulkEditListFilters/utils/optionsRecordIdentifiers.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+export const buildCheckboxFilterOptions = optionsMap => {
```

Comment:
Can be `buildCheckboxFilterOptions = ({label, value, disabled})=> {`
It will help simplify object